src/map.js

```diff
@@ -11,11 +11,7 @@ import transform from "./transform.js";
  * @param {string | string[] | function} [o.except] Ignore walking nodes of these types
  */
 export default function map (node, mappings, o) {
-	if (!Array.isArray(mappings)) {
-		mappings = [mappings];
-	}
-	mappings.push(
-		(node, property, parent, originalNode) => node === originalNode ? {...node} : node
-	);
+	const cloneFn = (node, property, parent, originalNode) => node === originalNode ? {...node} : node;
```

Comment:
```suggestion
	const cloneFn = (node, _, _, originalNode) => node === originalNode ? {...node} : node;
```

(It's a convention to use `_` for unused arguments — some linters even complain otherwise)

Reply:
For 2 unused args, is it a convention to use `_` and `__` (i.e. two underscores for the second one)? Or is there another thing we use for the second unused arg (otherwise it will complain about duplicate param name)

Reply:
Interesting, I've never come across the need to name a second one before and googling a bit, I can't find anything on it either, all examples involve a single argument 😅
Leave it then, though yet another reason to go with an object literal...

Reply:
Sounds good